client/webserver/passbytes.go

```diff
@@ -0,0 +1,188 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package webserver
+
+import (
+	"fmt"
+	"strconv"
+	"unicode"
+	"unicode/utf16"
+	"unicode/utf8"
+)
+
+const (
+	escapeSequence        = '\\'
+	unicodePrefix         = 'u'
+	unicodeSequenceLength = 6
+)
+
+// PassBytes is an alias of type []byte that implements json.UnmarshalJSON
+// to enable parsing json-encoded strings as []byte.
```

Comment:
Description is a little off. We're actually parsing JSON-encoded `[]byte` to utf-8 `[]byte` and skipping the `string` altogether, which is kind of the point.